paddle/fluid/operators/array_to_lod_tensor_op.cc

```diff
@@ -99,18 +102,24 @@ class ArrayToLoDTensorOp : public framework::OperatorBase {
     for (size_t i = 1; i < x.size(); ++i) {
       auto ins_i_dims = rank > 1 ? framework::slice_ddim(x[i].dims(), 1, rank)
                                  : framework::make_ddim({0});
-      PADDLE_ENFORCE_EQ(ins_i_dims, ins_dims,
-                        "The dimension of the %zu'th element in LoDTensorArray "
-                        "differs from previous ones.",
-                        i);
-      PADDLE_ENFORCE(x[i].place() == place,
-                     "The place class of the %zu'th element in LoDTensorArray "
-                     "differs from previous ones.",
-                     i);
-      PADDLE_ENFORCE(x[i].type() == data_type,
-                     "The date type of the %zu'th element in LoDTensorArray "
-                     "differs from previous ones.",
-                     i);
+      PADDLE_ENFORCE_EQ(
+          ins_i_dims, ins_dims,
+          platform::errors::InvalidArgument(
+              "The dimension of the %zu'th element in LoDTensorArray "
+              "differs from previous ones.",
+              i));
+      PADDLE_ENFORCE_EQ(
+          x[i].place(), place,
+          platform::errors::InvalidArgument(
+              "The place class of the %zu'th element in LoDTensorArray "
+              "differs from previous ones.",
+              i));
+      PADDLE_ENFORCE_EQ(
+          x[i].type(), data_type,
+          platform::errors::InvalidArgument(
+              "The date type of the %zu'th element in LoDTensorArray "
```

Comment:
give the data type, the current one's data type is *, the previous one's data type is *